content/_help/get-started/authentication-methods._en.md

```diff
@@ -5,17 +5,29 @@ category: get-started
 permalink: /help/get-started/authentication-methods/
 order: 2
 redirect_from:
+  - /help/creating-an-account/authentication-application/
+  - /help/changing-settings/how-do-I-set-up-an-authentication-app/
+  - /help/creating-an-account/two-factor-authentication/
+  - /help/creating-an-account/do-i-need-a-mobile-phone-to-use-logingov/
+  - /help/creating-an-account/what-is-two-factor-authentication/
   - /help/security-keys/does-using-a-security-key-mean-im-completely-safe-from-phishing/
   - /help/security-keys/what-is-a-security-key/
+  - /help/creating-an-account/personal-key/
```

Comment:
I'm not entirely sure what the old page said, and personal keys used to have a different purpose long ago, but there is a "Personal key" dedicated page:

https://login.gov/help/manage-your-account/personal-key/

Previously, personal keys used to work as an MFA method and didn't require identity verification, so I'm not positive the link above would be the best bet. Unless we can find a reference for what the old content contained, I might actually be included to keep it as you've implemented currently.

Reply:
It looks like the page for the old URL was about the same
https://web.archive.org/web/20201016160355/https://login.gov/help/creating-an-account/personal-key/

I'll move that from Authentication Methods to the Personal Key page. Thanks!